CHANGELOG.rst

```diff
@@ -142,7 +143,7 @@ Bug fixes
 - Both QT and GTK versions will reload hotkeys after a keymap change event.
 - Fix locking issue
 - Expose Alt_GR as a hotkey modifier on GTK.
-- (GTK) Fixed GUI lock-up, if multiple script error notifications are posted in quick succession. The notifications are now rate-limited and won’t post more than one notification per second. Fixes issue #383 
+- (GTK) Fixed GUI lock-up, if multiple script error notifications are posted in quick succession. The notifications are now rate-limited and won’t post more than one notification per second. Fixes issue #383
```

Comment:
Whitespace change. Ideally we wouldn't have trailing whitespace in the first place, but it is usually considered annoying to open PRs that change whitespace in areas that aren't related to your change.

(Only a small PR so doesn't make a big difference in this case, but keep it in mind. Editor tools sometimes do this automatically, so review your diffs)

Reply:
Ah dang it, this must have been introduced in the rebase. I can remove the change, so I will.